### ISSUE TITLE
Docstring of @property is not rendered correctly

### ISSUE THREAD

**Issue by yuxqiu** [microsoft/pylance-release#3888]
## Environment data

-   Language Server version: Pylance language server 2023.1.40
-   OS and version: macOS 13.2 (22D49)
-   Python version (& distribution if applicable, e.g. Anaconda): 3.10

## Code Snippet

```python
class Test:
    def __init__(self) -> None:
        self.__test = False

    @property
    def test(self):
        """Test DocString.

        Returns
        -------
        bool
            Lorem Ipsum
        """
        return self.__test
```

## Expected behavior

The pop-up window should show a brief description of the function and the return type.

## Actual behavior

<img width="433" alt="image" src="https://user-images.githubusercontent.com/45459572/215577565-4a23f8f1-7cab-40b3-8e01-07b94813b017.png">

The docstring is rendered twice in the pop-up.

It is also worth noting that this problem does not exist when using autocomplete.

<img width="654" alt="image" src="https://user-images.githubusercontent.com/45459572/215577097-0fbb7a61-8b9c-4ad5-8427-93d059b29fe6.png">

## Logs

The log after hovering over the `test` method.

```
(68851) [FG] parsing: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/typing.pyi [fs read 3ms] (35ms)
(68851) [FG] binding: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/typing.pyi (5ms)
(68851) [FG] parsing: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/typing_extensions.pyi [fs read 0ms] (3ms)
(68851) [FG] binding: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/typing_extensions.pyi (2ms)
(68851) [FG] parsing: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/_typeshed/__init__.pyi [fs read 0ms] (6ms)
(68851) [FG] binding: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/_typeshed/__init__.pyi (2ms)
(68851) [FG] parsing: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/types.pyi [fs read 0ms] (6ms)
(68851) [FG] binding: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/types.pyi (2ms)
(68851) [FG] parsing: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/abc.pyi [fs read 0ms] (1ms)
(68851) [FG] binding: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/abc.pyi (0ms)
(68851) [FG] parsing: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/collections/abc.pyi [fs read 0ms] (0ms)
(68851) [FG] binding: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/collections/abc.pyi ...
(68851) [FG]   parsing: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/_collections_abc.pyi [fs read 1ms] (1ms)
(68851) [FG]   binding: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/_collections_abc.pyi (0ms)
(68851) [FG] binding: /Users/x/.vscode/extensions/ms-python.vscode-pylance-2023.1.40/dist/typeshed-fallback/stdlib/collections/abc.pyi (2ms)
```

**Comment by heejaechang** [comment]
@bschnurr any idea?

**Comment by bschnurr** [comment]
I'll take a look

**Comment by bschnurr** [comment]
This issue has been fixed in prerelease version 2023.2.21, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023221-8-February-2023-PreRelease)
